arch/arm/boot/dts/overlays/README

```diff
@@ -862,6 +862,10 @@ Params: cam0-arducam-64mp       Select Arducam64MP for camera on port 0
         cam1-ov7251             Select OV7251 for camera on port 1
         cam1-ov9281             Select OV9281 for camera on port 1
         cam1-imx290-clk-freq    Set clock frequency for an IMX290 on port 1
+        cam0-sync-source        Set camera on port 0 as vsync source (for supported cameras)
```

Comment:
These lines are too long - I think you could drop the `(for supported cameras)` proviso.

Reply:
Right, I removed that now and also fixed the missing sign-off lines